LVT/LVT.Services/MeasurePresenter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using LVT.LVT.Interfaces;
+using System;
+using System.Collections.Generic;
+
+namespace LVT.LVT.Services
+{
+    public class MeasurePresenter : IVisualizer<Measure>
+    {
+        protected List<string[]> measureRowData = new List<string[]>();
+
+        public string VisualizeToString(Measure measure, string parentNode)
+        {
+            return "[{ v: '" + measure.NodeID + "', f: 'Measure<div style=\"font-style:italic\">" + $"{measure.Description}" + "</div>" +
+                                                                                   "<div style=\"font-style:italic\">" + $"{measure.Deadline}" + "</div>" +
+                                                                                   "<div style=\"font-style:italic\">" + $"{measure.Amount}" + "</div>" +
+                                                                                   "<div style=\"font-style:italic\">" + $"{measure.Units}" + "</div>" +
+                                                                                   "'}, " + $"'{parentNode}']";
```

Comment:
This is one hell of a lot of hard coding in the line, see other comments on other presenter files

Reply:
as above.  Not a fan of hardcoding as much as I've been doing here, but other than loads of refactoring (as discussed above) I'm not sure how easily this can be fixed now.